tests/DomTestCaseTest.php

```diff
@@ -1008,6 +1008,7 @@ public function testAssertXPathCountPresentTrue()
     /**
      * @covers            PHPUnit_Framework_DOMTestCase::assertXPathCount
      * @expectedException PHPUnit_Framework_AssertionFailedError
+     * @expectedExceptionMessage Failed asserting that false is true.
```

Comment:
Yes, currently I'm just using the default exception messages thrown by phpunit assert methods. We should use better defaults when ones aren't supplied by the caller.